packages/portal-proto/src/components/Modals/tests/SaveOrCreateEntityModal.unit.test.tsx

```diff
@@ -1,13 +1,11 @@
-import { MantineProvider } from "@mantine/core";
```

Comment:
Could you add a test for displaying the unsaved_cohort warning?

Reply:
just updated.

Reply:
Cool, great coverage! Could you change the names on the tests to be a little more descriptive? Like "disallowed names with whitespace are not accepted" instead of "disallowed names are not accepted - VERSION 2".

Reply:
This is done :)